### DISCUSSION TITLE
Servcices Missing

### DISCUSSION THREAD

**Discussion by jeffthedog** [Security-Onion-Solutions/securityonion#13343] [2.4]
### Version

2.4.80

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Eval

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

8

### RAM

10347MB

### Storage for /

-

### Storage for /nsm

-

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

Hey guys,

been working on setting up my security onion with the ISO image on my Virtual Machine (virtual box) 

I am unable to see the webpage and realised it is due to this

<img width="377" alt="image" src="https://github.com/user-attachments/assets/0116dc77-9539-4b07-b671-bdc7b9b83556">
Any suggestions on what can be done?

Thanks

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment]
What kind of storage are you using (NVME, SSD, or rotational)?

What are the specs of the host machine?

Are there other virtual machines or processes running?

Are you able to increase the RAM in the VM to see if that makes any difference?

You specified above that you are sniffing network traffic at 1Gbps to 10Gbps. If you are trying to sniff live traffic at 1 gigabit or more, then you will need more than the minimum hardware requirements:
https://docs.securityonion.net/en/2.4/hardware.html#minimum-specs
https://docs.securityonion.net/en/2.4/hardware.html#sensor-hardware-considerations
